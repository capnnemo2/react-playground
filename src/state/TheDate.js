import React from 'react';

export class TheDate extends React.Component {
    render() {
        return(
            <div />
        )
    }
}