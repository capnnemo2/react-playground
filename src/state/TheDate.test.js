import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {TheDate} from './TheDate';

describe('TheDate Component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<TheDate />, div);
        ReactDOM.unmountComponentAtNode(div);
    })

    // it doesn't match the snapshot because the time always changes. it.skip for now...
    it.skip('renders the UI as expected', () => {
        expect(
            renderer.create(<TheDate />).toJSON()
        ).toMatchSnapshot()
    })
})