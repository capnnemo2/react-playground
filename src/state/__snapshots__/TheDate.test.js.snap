// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDate Component renders the UI as expected 1`] = `
<div>
  1/15/2020, 3:10:43 PM
</div>
`;
